tests/unit/src/test/scala/tests/FormattingLspSuite.scala

```diff
@@ -316,7 +316,8 @@ class FormattingLspSuite extends BaseLspSuite("formatting") {
       _ = {
         assertNoDiff(
           client.workspaceMessageRequests,
-          MissingScalafmtVersion.messageRequestMessage,
+          s"""|${MissingScalafmtVersion.messageRequestMessage}
```

Comment:
This changed because of the additional message from slow task, which was actually never invoked in tests.